runtime/src/main/java/org/corfudb/protocols/wireprotocol/CorfuMsgType.java

```diff
@@ -60,6 +60,10 @@
     READ_RESPONSE(32, new TypeToken<CorfuPayloadMsg<ReadResponse>>() {}),
     MULTIPLE_READ_REQUEST(35, new TypeToken<CorfuPayloadMsg<MultipleReadRequest>>() {}),
     PREFIX_TRIM(38, new TypeToken<CorfuPayloadMsg<TrimRequest>>() {}),
+
+    LOG_FILE_SIZE_REQUEST(39, TypeToken.of(CorfuMsg.class)),
+    LOG_FILE_SIZE_RESPONSE(40, new TypeToken<CorfuPayloadMsg<Long>>(){}),
```

Comment:
I would make this a little more generic. We need an API to get LogUnit metrics and the log size an be one of those stats.